docs/source/markdown/podman-image-inspect.1.md

```diff
@@ -18,6 +18,34 @@ all results in a JSON array.  If a format is specified, the given template will
 Format the output using the given Go template.
 The keys of the returned JSON can be used as the values for the --format flag (see examples below).
 
+Valid placeholders for the Go template are listed below:
+
+| **Placeholder**      | **Description**                                    |
+| -----------------    | ------------------                                 |
+| .Annotations         | Annotation information included in the image       |
+| .Architecture        | Architecture of software in the image              |
+| .Author              | Image author                                       |
+| .Comment             | Image comment                                      |
+| .Config ...          | Structure with config info                         |
+| .Created             | Image creation time (string, ISO3601)              |
+| .Digest              | Image digest (sha256:+64-char hash)      	    |
```

Comment:
There's a tab character in here.

I don't know if it's something in my emacs setup, or if it's automatic in emacs, but when I have my cursor inside MD tables and press the TAB key, emacs instantly aligns all the vertical bars. Really sweet. HTH.

Reply:
Must be something in your emacs, would love to have that.

Reply:
Must be this [markdown-mode](https://jblevins.org/projects/markdown-mode/) that I have installed (unfortunately, not in Fedora repos).